tutorials/documentation_materials/classiq_101/classiq_concepts/execute/advanced_execution/advanced_execution_user_guide.ipynb

```diff
@@ -0,0 +1,834 @@
+{
```

Comment:
<p>A few things that should be made clearer:</p><p> </p><ul><li>what does it mean "with two varying parameters"?-  this confuses me a bit</li><li>Estimetate - calculates the expectation value and the variance of a specified operator for the quantum state determined by the quantum program,</li><li>What is the "Submit" - if it is stated here, it must be explained briefly. </li></ul><p> </p><p>It is good that you survey what's going to be done in the tutorial, but currently it raises many questions. Please elaborate the different points brifley so they can be understood for people that haven't used ExecutionSession before.</p><p> </p><p>Also, is it relevant only for the SDK? if so, also should be stated</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/608/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>I hope it clearer now.</p>